README.md

```diff
@@ -35,22 +34,27 @@ if __name__ == '__main__':
         resized = contents.resize(0.5)
         write_image(output_folder / file.name)
 ```
-Doing that at scale can take a long time. **We'd like to run that script in AzureML, consume the data from a folder in
-blob storage, and write the results back to blob storage**.
+Doing that at scale can take a long time. We'd like to run that script in AzureML, consume the data from a folder in
+blob storage, and write the results back to blob storage.
 
-With the `hi-ml` package, you can turn that script into one that runs on the cloud by adding one function call:
+Pre-requisite: Download the config file from your AzureML workspace, as described 
+[here](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-configure-environment). Put this file into
+the folder where your script lives.
 
 ```python
 from pathlib import Path
 from health.azure.himl import submit_to_azure_if_needed
 if __name__ == '__main__':
     current_file = Path(__file__)
-    run_info = submit_to_azure_if_needed(compute_cluster_name="preprocess-ds12",
+    run_info = submit_to_azure_if_needed(entry_script=current_file, 
+                                         snapshot_root_directory=current_file.parent,
+                                         workspace_config_path=Path("config.json"),
+                                         compute_cluster_name="preprocess-ds12",
+                                         conda_environment_file=Path("environment.yml"),
                                          input_datasets=["images123"],
                                          # Omit this line if you don't create an output dataset (for example, in
                                          # model training scripts)
-                                         output_datasets=["images123_resized"],
-                                         default_datastore="my_datastore")
+                                         output_datasets=["images123_resized"],)
```

Comment:
I cleaned that up to reduce clutter in a previous PR, Is there an important reason why you put that back in?